internal/config/config.go

```diff
@@ -221,6 +251,41 @@ func (c *SingleChainConfig) isValid() bool {
 	return isChainConfigValid
 }
 
+func (r *RoutingConfig) setDefaults() {
+	if r.Errors == nil || r.Latency == nil {
```

Comment:
Should this be `&&`? You should be able to define error routing but not latency routing (and vise versa)